advanced_functionality/r_forecast_bring_your_own/R Forecast Bring-Your-Own Container Demo.ipynb

```diff
@@ -0,0 +1,333 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# R Forecastig Bring-Your-Own Container Demo\n",
+    "\n",
+    "This notebook demonstrates how to use the container to create an endpoint serving\n",
+    "forecasts made by the [R forecast package](https://cran.r-project.org/web/packages/forecast/index.html) using Amazon SageMaker."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import time\n",
+    "import urllib\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline\n",
+    "import sagemaker"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# set this to the ECR image url of the container, returned by build_and_push.sh\n",
+    "# e.g. 123456789.dkr.ecr.us-east-1.amazonaws.com/r_forecast_bring_your_own:latest\n",
+    "CONTAINER_IMAGE = \"to_be_set\" \n",
+    "\n",
+    "EXECUTION_ROLE_ARN = sagemaker.get_execution_role()  # or set manually\n",
+    "\n",
+    "INSTANCE_TYPE = \"ml.c5.xlarge\"\n",
+    "INSTANCE_COUNT = 1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sagemaker_session = sagemaker.Session()\n",
+    "sagemaker_client = sagemaker_session.sagemaker_client\n",
+    "sagemaker_runtime_client = sagemaker_session.sagemaker_runtime_client\n",
+    "\n",
+    "# Region for SageMaker calls -- should be the same as your ECR\n",
+    "print(\"Region: \" + sagemaker_session.boto_region_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Endpoint Creation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create endpoint from container image\n",
```

Comment:
Please be consistent with the spacing. There are some cells that have a newline in between the comment and some that don't.